examples/web-sdk-sample/README.md

```diff
@@ -0,0 +1,38 @@
+# Clickstream web SDK sample project
+
+Creating a monorepo project with Vue2.x and Express.JS using Yarn Workspaces for Clickstream Analytics on AWS solution web SDK sample.
+
+Read the article: [Yarn Workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/)
+
+## Project structure
+
+We have 2 packages inside the project:
+
+- **Vue2:** Vue2.x application.
+- **Server:** Express.JS application.
+
+Each of the packages have their own `package.json` file, so they define their dependencies as well as they have fully autonomy to publish a new version of the package into NPM or Yarn registry.
+
+```
+|- package.json => root workspace (private package used by yarn workspaces)
+|--- packages
+|------ vue2
+|-------- package.json  => Vue2.x project
+|------ server
+|-------- package.json => Express.js project
+```
+
+## How to install and execute
+
+> Important! The node version for the project is 16. Make sure you have that version installed in your computer. If you have NVM installed, run `nvm use 16`. If not, install it here: https://github.com/nvm-sh/nvm#install--update-script
```

Comment:
Why we must stick to Node 16 which is end of support. How about using new LTS Node 18 or Node 20.

Reply:
We won't depend on node 16. Let's rewrite this.